src/xcode/ENA/ENAUITests/ExposureSubmission/ENAUITests_04a_ExposureSubmission.swift

```diff
@@ -249,8 +249,68 @@ class ENAUITests_04a_ExposureSubmission: CWATestCase {
 		app.cells[AccessibilityIdentifiers.Home.activateCardOnTitle].waitAndTap()
 
 	}
+	
+	func test_SRSFlow_PositiveWithoutResultInTheApp() {
+		app.setLaunchArgument(LaunchArguments.common.ENStatus, to: ENStatus.active.stringValue)
+		launch()
+		
+		// Start Submission Flow
+		app.cells.buttons[AccessibilityIdentifiers.Home.submitCardButton].waitAndTap()
+		
+		// Start Selftest Flow with Positive Test Without Result In The App
+		app.buttons[AccessibilityIdentifiers.ExposureSubmissionDispatch.SRSButtonDescription].waitAndTap()
+		
+		// SRS Consent Screen And Tap Next
+		XCTAssertTrue(app.cells[AccessibilityIdentifiers.SRSConsentScreen.acknowledgementTitle].waitForExistence(timeout: .medium))
+		app.buttons[AccessibilityIdentifiers.ExposureSubmissionTestResultAvailable.primaryButton].waitAndTap(.long)
 
-	/* Commented out in favour of SRS story
+		// Type of Tests Screen is showing
+		XCTAssertTrue(
+			app.staticTexts[AccessibilityIdentifiers.ExposureSubmission.SRSTestTypeSelection.body]
+				.waitForExistence(timeout: .medium)
+		)
+
+		// Select Self Test and Tap Next
+		app.buttons[AccessibilityIdentifiers.ExposureSubmission.SRSTestTypeSelection.optionSRSSelfTest].waitAndTap()
+		app.buttons[AccessibilityIdentifiers.General.primaryFooterButton].waitAndTap()
+		
+		// Select Symptoms Screen is showing
+		XCTAssertTrue(
+			app.staticTexts[AccessibilityIdentifiers.ExposureSubmissionSymptoms.description]
+				.waitForExistence(timeout: .medium)
+		)
+		
+		// Select Symptoms Yes And Tap Next
+		app.buttons[AccessibilityIdentifiers.ExposureSubmissionSymptoms.answerOptionYes].waitAndTap()
+		app.buttons[AccessibilityIdentifiers.ExposureSubmission.primaryButton].waitAndTap()
+		
+		// Symptoms Onset Screen is showing
+		XCTAssertTrue(
+			app.staticTexts[AccessibilityIdentifiers.ExposureSubmissionSymptomsOnset.subtitle]
+				.waitForExistence(timeout: .medium)
+		)
+		
+		// Select Option Last Seven Days And Tap Next
+		app.buttons[AccessibilityIdentifiers.ExposureSubmissionSymptomsOnset.answerOptionLastSevenDays].waitAndTap()
+		app.buttons[AccessibilityIdentifiers.ExposureSubmission.primaryButton].waitAndTap()
+		
+		// Consent Warn Others Alert Appears
+		XCTAssertTrue(
+			app.alerts.firstMatch.waitForExistence(timeout: .medium)
+		)
+
+		// Confirm Warn Others
+		app.alerts.firstMatch.buttons.element(boundBy: 0).waitAndTap()
+		
+		// SRS Submission done and show Thank You Screen
+		XCTAssertTrue(
+			app.staticTexts[AccessibilityIdentifiers.ExposureSubmissionSuccess.description]
+				.waitForExistence(timeout: .medium)
+		)
+	}
+
+	/*
+	// Commented out in favour of SRS story
```

Comment:
Perhaps we need another task to fix the existing UI tests which we commented out. We need to find another way to mock the tests as we have removed TAN.

Reply:
Yes, I would also suggest to move that part into another task.